README.md

```diff
@@ -21,17 +21,30 @@ from trycourier import Courier
 
 client = Courier(auth_token="your-auth-token") #or set via COURIER_AUTH_TOKEN env var
 
-resp = client.send(
-    event="your-event-id",
-    recipient="your-recipient-id",
-    profile={
-        "email": "example@example.com",
-        "phone_number": "555-867-5309"
+resp = client.send_message(
```

Comment:
do we want to replace the old example or add v2 calls as new examples? I think we should keep old calls for existing customers who already have been using SDK for v1 requests so if they need to read thru README v1 examples are handy.